hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -211,6 +214,7 @@
   private PipelineChoosePolicy pipelineChoosePolicy;
 
   private HDDSLayoutVersionManager scmLayoutVersionManager;
+  private UpgradeFinalizer upgradeFinalizer;
```

Comment:
Nit. Can be UpgradeFinalizer\<StorageContainerManager\>.

Reply:
yup. Made the changes as suggested.